src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+promise1
+  .then((message) => {
+    const successMessage = document.createElement('div');
+
+    successMessage.classList.add('message');
+    successMessage.textContent = 'Promise was resolved!';
+    document.body.appendChild(successMessage);
+  })
+  .catch(() => {
+    const errorMessage = document.createElement('div');
+
+    errorMessage.classList.add('message');
+    errorMessage.classList.add('error-message');
+    errorMessage.textContent = 'Promise was rejected!';
+    document.body.appendChild(errorMessage);
+  });
```

Comment:
The catch block for promise1 is redundant because the promise is only resolved when the logo is clicked. Consider removing it or handling specific errors if needed.